ts/Extensions/Autoload/DependencyMapping.ts

```diff
@@ -0,0 +1,1076 @@
+    ],
+    'plotOptions.boxplot.dragDrop.draggableQ1': [
+        'modules/draggable-points'
+    ],
+    'plotOptions.boxplot.dragDrop.draggableQ3': [
+        'modules/draggable-points'
+    ],
+    'plotOptions.boxplot': [
+        'highcharts-more'
+    ],
+    'plotOptions.bubble': [
+        'highcharts-more'
+    ],
+    'plotOptions.bullet.dragDrop.draggableTarget': [
+        'modules/draggable-points'
+    ],
+    'plotOptions.bullet': [
+        'modules/bullet'
+    ],
+    'plotOptions.cci': [
+        'stock/indicators/indicators',
+        'stock/indicators/cci'
+    ],
+    'plotOptions.chaikin': [
+        'stock/indicators/indicators',
+        'stock/indicators/chaikin'
+    ],
+    'plotOptions.cmf': [
+        'stock/indicators/indicators',
+        'stock/indicators/cmf'
+    ],
+    'plotOptions.cmo': [
+        'stock/indicators/indicators',
+        'stock/indicators/cmo'
+    ],
+    'plotOptions.column.depth': [
+        'highcharts-3d'
+    ],
+    'plotOptions.column.edgeColor': [
+        'highcharts-3d'
+    ],
+    'plotOptions.column.edgeWidth': [
+        'highcharts-3d'
+    ],
+    'plotOptions.column.groupZPadding': [
+        'highcharts-3d'
+    ],
+    'plotOptions.columnpyramid': [
+        'highcharts-more'
+    ],
+    'plotOptions.columnrange.dragDrop.draggableHigh': [
+        'modules/draggable-points'
+    ],
+    'plotOptions.columnrange.dragDrop.draggableLow': [
+        'modules/draggable-points'
+    ],
+    'plotOptions.columnrange': [
+        'highcharts-more'
+    ],
+    'plotOptions.cylinder': [
+        'modules/cylinder'
+    ],
+    'plotOptions.dema': [
+        'stock/indicators/indicators',
+        'stock/indicators/dema'
+    ],
+    'plotOptions.dependencywheel': [
+        'modules/dependency-wheel'
+    ],
+    'plotOptions.disparityindex': [
+        'stock/indicators/indicators',
+        'stock/indicators/disparity-index'
+    ],
+    'plotOptions.dmi': [
+        'stock/indicators/indicators',
+        'stock/indicators/dmi'
+    ],
+    'plotOptions.dpo': [
+        'stock/indicators/indicators',
+        'stock/indicators/dpo'
+    ],
+    'plotOptions.ema': [
+        'stock/indicators/indicators'
+    ],
+    'plotOptions.errorbar': [
+        'highcharts-more'
+    ],
+    'plotOptions.flowmap': [
+        'modules/flowmap'
+    ],
+    'plotOptions.funnel': [
+        'modules/funnel'
+    ],
+    'plotOptions.funnel3d': [
+        'highcharts-3d',
+        'modules/cylinder',
+        'modules/funnel3d'
+    ],
+    'plotOptions.gantt.dragDrop.draggableEnd': [
+        'modules/draggable-points'
+    ],
+    'plotOptions.gantt.dragDrop.draggableStart': [
+        'modules/draggable-points'
+    ],
+    'plotOptions.gantt': [
+        'highcharts-gantt'
+    ],
+    'plotOptions.gauge': [
+        'highcharts-more'
+    ],
+    'plotOptions.geoheatmap': [
+        'modules/geoheatmap'
+    ],
+    'plotOptions.heikinashi': [
+        'modules/heikinashi'
```

Comment:
Not sure if a problem, but if we create a chart with Chart() constructor, then Stock won't be loaded and heikinashi will throw errors, tested with:

```ts
Highcharts.chart('container', {
    series: [{
        data: [
            [0, 1, 1],
            [1, 2, 4],
            [2, 5, 3],
            
            [3, 4, 5]
        ],
        type: 'heikinashi',
        pointIntervalUnit: 'year'
    }],
    accessibility: {
        enabled: true
    },
    exporting: {
        enabled: true
    }
});

```